pyproject.toml

```diff
@@ -229,6 +227,7 @@ ignore = [
     "D105", # undocumented-magic-method
     "D106", # undocumented-public-nested-class
     "D107", # undocumented-public-init
+    "COM812" # missing-trailing-comma -- disabled for compatability with formatter.
```

Comment:
Wait, don't you have trailing commas now? Should this ignore be here?

Reply:
Yes, otherwise we get this warning:

```
The following rules may cause conflicts when used with the formatter: `COM812`. To avoid unexpected behavior, we recommend disabling these rules, either by removing them from the `select` or `extend-select` configuration, or adding them to the `ignore` configuration.
```

See the new comment's attached issue.

Reply:
The important part is that the formatter will add the commas so we effectively get the same behavior.